### ISSUE TITLE
Unable to download packages (503 Server Error)

### ISSUE THREAD

**Issue by jakirkham** [ContinuumIO/anaconda-issues#583]
HTTPError: 503 Server Error: Service Unavailable: Back-end server is at capacity for url: https://repo.continuum.io/pkgs/pro/noarch/repodata.json.bz2: https://repo.continuum.io/pkgs/pro/noarch/

**Comment by goanpeca** [comment]
:+1: just clicked on both, and they seem to be fine

**Comment by jakirkham** [comment]
That's true also retried moments before you wrote that. :) Seems to have changed.

**Comment by jakirkham** [comment]
It's back.

**Comment by jakirkham** [comment]
Now it is gone. It would be nice if this intermittent timeout could be understood. I'm leaving this open for that reason.

**Comment by jakirkham** [comment]
Yet again this is happening. Only noting this so it is clear how frequently this problem is arising (i.e. 15mins later).

**Comment by groutr** [comment]
I can bring this to the attention of our IT folks on Monday.  Hopefully it's not a trend.

**Comment by jakirkham** [comment]
Lately, I have been seeing a 404 error that appears sporadically. It normally goes away with a retry, but it has been happening for a little while now.

**Comment by edmondja** [comment]
Hello, i have exactly the same problem right now.

**Comment by eroesch** [comment]
Jep, me too. How come does it need to hook up with a remote back-end? Surely, it should be able to run without an internet connection, no?

**Comment by allanelder** [comment]
I'm getting the same issue.

**Comment by allanelder** [comment]
OK.. seems to be back up now, guess it was a glitch in the matrix!

**Comment by akleeman** [comment]
Seems this same issue is back again.  I've been getting it on and off the last couple days.

Here's an example error:
```Error: HTTPError: 503 Server Error: Service Unavailable: Back-end server is at capacity: https://repo.continuum.io/pkgs/free/linux-64/openssl-1.0.2h-0.tar.bz2```

Sometimes it happens when trying to update conda

```
conda update -q conda
...
Error: HTTPError: 503 Server Error: Service Unavailable: Back-end server is at capacity: https://repo.continuum.io/pkgs/free/linux-64/conda-env-2.4.5-py27_0.tar.bz2
```

I can typically restart the build (this is happening in a Travis build) but lately I've had to restart builds 3/4 times before it'll run through clean.

Anything I can do on my end to make it more robust?

**Comment by MattF-NSIDC** [comment]
Have been experiencing this issue for the last 20 minutes or so.

**Comment by VasilyTokarev** [comment]
same

**Comment by jakirkham** [comment]
Us too.

**Comment by MattF-NSIDC** [comment]
Looking good now!

**Comment by Korijn** [comment]
We're seeing this occur regularly for the past few days now. It's becoming more frequent. Could it have something to do with conda now cleaning out the cache automatically at certain conditions?

**Comment by frol** [comment]
It hit our servers also! We have been experiencing this issue for over 24 hours now.
